<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1677</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/icon.jpg</normaloff>:/images/icon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">#centralWidget{
	
	border-image: url(:/MainWindow/mylibrary.jpg);
  color:white;
}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>-500</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label{
color:white;

}</string>
    </property>
    <property name="text">
     <string>REGISTER AND LOGIN FORM</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="loginForm">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>170</y>
      <width>271</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#loginForm{
	
 background: url(:/MainWindow/logbg.png) no-repeat center center fixed;
border-radius:8px;
color:white;

}</string>
    </property>
    <property name="title">
     <string>Login</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>251</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="usernameLogin">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#usernameLogin
{
color:white;
}
</string>
          </property>
          <property name="text">
           <string>Username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="username_2">
          <property name="styleSheet">
           <string notr="true">#username_2
{
background-color: rgb(56, 40, 30);
color:white;
}
</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="passwordLogin">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#passwordLogin
{
color:white;
}
</string>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="password_2">
          <property name="styleSheet">
           <string notr="true">#password_2
{
background-color: rgb(56, 40, 30);
color:white;
}
</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="LogInBtn">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#LogInBtn{
color:white;
background-color:rgb(88, 54, 34);

border-width:0px;
border-radius:4px;
}

#LogInBtn:hover{
background-color:rgb(99, 60, 52);
}</string>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="registerForm">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>291</width>
      <height>511</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#registerForm{

 background: url(:/MainWindow/logbg.png) no-repeat center center fixed;
border-radius:8px;
color:white;

}</string>
    </property>
    <property name="title">
     <string>Sign up</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>261</width>
       <height>451</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#label_2
{
color:white;
}
</string>
          </property>
          <property name="text">
           <string>Username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="username">
          <property name="styleSheet">
           <string notr="true">#username{
	background-color: rgb(56, 40, 30);
color:white;
}
</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#label_3
{
color:white;
}
</string>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="password">
          <property name="styleSheet">
           <string notr="true">#password
{
	background-color: rgb(56, 40, 30);
color:white;
}
</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#label_4
{
color:white;
}
</string>
          </property>
          <property name="text">
           <string>First Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="firstName">
          <property name="styleSheet">
           <string notr="true">#firstName{
	background-color: rgb(56, 40, 30);
color:white;
}
</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#label_5
{
color:white;
}
</string>
          </property>
          <property name="text">
           <string>Last Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lastName">
          <property name="styleSheet">
           <string notr="true">#lastName{
	background-color: rgb(56, 40, 30);
color:white;
}
</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SignUpBtn">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#SignUpBtn{
color:white;
background-color:rgb(88, 54, 34);

border-width:0px;
border-radius:4px;
}

#SignUpBtn:hover{
background-color:rgb(99, 60, 52);
}</string>
          </property>
          <property name="text">
           <string>Sign Up</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
